db/seeds.rb

```diff
@@ -79,15 +79,54 @@ def position(type)
   surname = player['web_name']
   full_name = "#{player['first_name']} #{player['second_name']}"
   position = position(player['element_type'])
+
+   # Get specific player url
+   url = 'https://fantasy.premierleague.com/drf/element-summary/' << player['id'].to_s
+   response = HTTParty.get(url)
+ 
+    # Get specific player url
```

Comment:
you're getting the url ans response twice
✂️

Reply:
Updated